api.md

```diff
@@ -270,6 +295,12 @@ if `conditions` is already a hypertable:
 SELECT create_hypertable('conditions', 'time', if_not_exists => TRUE);
 ```
 
+Convert table `conditions` to hypertable with adaptive chunking
+enabled and a chunk target size of `2GB`:
+```sql
+SELECT create_hypertable('conditions', 'time', chunk_target_size => '2GB');
+```
+
 #### Best Practices [](create_hypertable-best-practices)
 
 Users of TimescaleDB often have two common questions:
```

Comment:
Best practices start with a paragraph that needs to be changed:

"Time intervals: The current release of TimescaleDB does not perform adaptive time intervals (although this is in the works). So, users must configure it when creating their hypertable by setting the chunk_time_interval (or use the default of 1 month). The interval used for new chunks can be changed by calling set_chunk_time_interval."

Perhaps rewrite to:

"Time intervals: The current release of TimescaleDB enables both the manual and automated adaption of its time intervals.   With manually-set intervals, users should configure the `chunk_time_interval` when creating their hypertable (the default value is 1 week).   The interval used for new chunks can be changed by calling set_chunk_time_interval.  With automatically adapted intervals (which are *not* enabled by default), the chunk interval will start at 1 day (unless otherwise specified at hypertable creation) and the algorithm will automatically adapt the interval for future chunks.  

The following instructions apply if users are choosing the configure time intervals themselves."